packages/ppx/test/css-support/css-support.expected.re

```diff
@@ -349,7 +349,7 @@ CssJs.cursor(`verticalText);
 CssJs.cursor(`alias);
 CssJs.cursor(`copy);
 CssJs.cursor(`noDrop);
-CssJs.cursor(`notAllowed);
```

Comment:
This can be tweaked to not conflict with `NOT_OPERATOR`, does that make sense?

Reply:
It doesn't actually, the not should be parsed in the media prelude, not on the component_value

Reply:
Make sense. A hack AFAIU is not to produce `NOT_OPERATOR`, instead we use `IDENT` in the parser which makes the `not` operator unsafe.

Reply:
no, parsing should remain as what It was before (2 separate component_values) and those errors should dissapear